cohort_4/playbook/ACTIVITIES.md

```diff
@@ -647,6 +647,40 @@ Here are good example failures that haven't been resolved:
 
 # Feedback Forms
 
+At the end of each day we want to collect feedback from the hackers to see what
+did/didn't do well and understand how the hackers feel about camp as it
+progresses. The feedback may also be helpful to diagnose issues with certain
+parts of camp (ex. the second workshop was particularly difficult and many
+people got lost and felt frustrated the day after).
+
+When giving the feedback forms to the hackers, it is _very important_ that the
+hackers feel that giving feedback is important. We don't want them to feel like
+they're filling out the forms to appease us. Rather, we want them filling out
+the forms because they want camp to be as good as possible and they know that
+giving valuable feedback improves camp for everyone.
+
+Making hackers feel like giving feedback is important can be broken down into
+two parts:
+
+1. Telling them why we want their feedback
+2. Actually acting on their feedback and _showing_ that it makes a difference
+
+When it comes time to send out feedback forms to everyone, we recommend saying
+something along the lines of:
+
+> Hey everyone! We're going to send out a link to a feedback form for today. We
+> want to do _everything we can_ to make camp as good as possible and we can't
+> do this without you.
+>
+> Please be as harsh as critical as you feel comfortable, we are going to do
```

Comment:
Good catch, will fix when I get back to the office.

Can you ping @gemmabusoni in real life once this is merged? I want to make sure this is accounted for in today's feedback.

--
Zach Latta
http://zachlatta.com
https://github.com/zachlatta
https://twitter.com/zachlatta

On Aug 4, 2015, 12:12 PM -0700, Max Wofford<notifications@github.com>, wrote:
>  
> Incohort_4/playbook/ACTIVITIES.md(https://github.com/hackedu/hack-camp/pull/314#discussion_r36229250):
>  
> >+giving valuable feedback improves camp for everyone.>+>+Making hackers feel like giving feedback is important can be broken down into>+two parts:>+>+1. Telling them why we want their feedback>+2. Actually acting on their feedback and _showing_ that it makes a difference>+>+When it comes time to send out feedback forms to everyone, we recommend saying>+something along the lines of:>+>+>Hey everyone! We're going to send out a link to a feedback form for today. We>+>want to do _everything we can_ to make camp as good as possible and we can't>+>do this without you.>+>>+>Please be as harsh as critical as you feel comfortable, we are going to do
>  
> harshANDcritical
>  
>  
> —
> Reply to this email directly orview it on GitHub(https://github.com/hackedu/hack-camp/pull/314/files#r36229250).
>  
>  
>